mula/scheduler/rankers/deadline.py

```diff
@@ -0,0 +1,41 @@
+import random
+from datetime import datetime, timedelta, timezone
+from typing import Any
+
+from scheduler.utils import cron
+
+from .ranker import Ranker
+
+
+class DefaultDeadlineRanker(Ranker):
+    def rank(self, obj: Any) -> int:
```

Comment:
So semantically the rank will now be a number that represents a timestamp equalling the "deadline" right? I believe that's what we discussed also. Meaning we will can infer that a boefje should run now because the current timestamp is really close to or even greater than the deadline?

Reply:
Correct. However, I can see that this can be confusing. In the case of deadline calculation we're not ranking but we're scheduling a job in the future. When the deadline has passed the job will be evaluated, ranked and then be pushed on to the queue.

Would you also agree to refactor/restructure this outside of the rankers module?